src/main/io/rcdevice_cam.c

```diff
@@ -229,6 +229,10 @@ static void rcdevice5KeySimulationProcess(timeUs_t currentTimeUs)
         return;
     }
 
+    if (getArmingDisableFlags() & ARMING_DISABLED_RUNAWAY_TAKEOFF) {
```

Comment:
The code would be a bit easier to read if this was combined with the ` if (ARMING_FLAG(ARMED))` condition above.